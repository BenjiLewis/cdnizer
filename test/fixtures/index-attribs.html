<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">

	<link rel="stylesheet" href="css/quoted-attrib-before.css">
	<link href="css/quoted-attrib-after.css" rel="stylesheet">
	<link rel="stylesheet" href=css/unquoted-attrib-after.css>
	<link href=css/unquoted-attrib-after.css rel=stylesheet>

	<link rel="stylesheet" href="css/endtag-quoted-attrib-before.css"></link>
	<link href="css/endtag-quoted-attrib-after.css" rel="stylesheet"> </link>
	<link rel="stylesheet" href=css/endtag-unquoted-attrib-after.css> </link>
	<link href=css/endtag-unquoted-attrib-after.css rel=stylesheet> </link>

</head>
<body>

<script src="js/quoted-attrib-after.js" type="text/javascript"></script>
<script type="text/javascript" src="js/quoted-attrib-before.js"></script>
<script src=js/unquoted-attrib-after.js type=text/javascript></script>
<script type=text/javascript src=js/unquoted-attrib-before.js></script>

<img src="img/quoted-attrib-after.jpg" width="12" height="12" />
<img width="12" height="12" src="img/quoted-attrib-before.jpg" />
<img src=img/unquoted-attrib-after.jpg width=12 height=12 />
<img width=12 height=12 src=img/unquoted-attrib-before.jpg />

</body>
</html>
