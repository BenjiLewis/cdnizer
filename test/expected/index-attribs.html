<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">

	<link rel="stylesheet" href="//examplecdn/css/quoted-attrib-before.css">
	<link href="//examplecdn/css/quoted-attrib-after.css" rel="stylesheet">
	<link rel="stylesheet" href=//examplecdn/css/unquoted-attrib-after.css>
	<link href=//examplecdn/css/unquoted-attrib-after.css rel=stylesheet>

	<link rel="stylesheet" href="//examplecdn/css/endtag-quoted-attrib-before.css"></link>
	<link href="//examplecdn/css/endtag-quoted-attrib-after.css" rel="stylesheet"> </link>
	<link rel="stylesheet" href=//examplecdn/css/endtag-unquoted-attrib-after.css> </link>
	<link href=//examplecdn/css/endtag-unquoted-attrib-after.css rel=stylesheet> </link>

</head>
<body>

<script src="//examplecdn/js/quoted-attrib-after.js" type="text/javascript"></script>
<script type="text/javascript" src="//examplecdn/js/quoted-attrib-before.js"></script>
<script src=//examplecdn/js/unquoted-attrib-after.js type=text/javascript></script>
<script type=text/javascript src=//examplecdn/js/unquoted-attrib-before.js></script>

<img src="//examplecdn/img/quoted-attrib-after.jpg" width="12" height="12" />
<img width="12" height="12" src="//examplecdn/img/quoted-attrib-before.jpg" />
<img src=//examplecdn/img/unquoted-attrib-after.jpg width=12 height=12 />
<img width=12 height=12 src=//examplecdn/img/unquoted-attrib-before.jpg />

</body>
</html>
